releasenotes/notes/url-normalization.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+releaseNotes:
+  - |
+    **Added** workload request URL path normalization configurations
```

add for sidecar proxy and gateways?  
```suggestion
    **Added** workload request URL path normalization configurations for sidecar proxy and gateways
```